src/resources/authoring/gv2.json

```diff
@@ -283,6 +283,7 @@
         "alleles": "w-W, m-m, T-T, h-h, A1-A1, C-C, B-B, D-D, rh-rh, Bog-Bog",
         "sex": 0
       },
+      "gametes": [[{}, {}, {}, {}, {}, {}, {}, {}], [{}, {}, {}, {}, {}, {}, {}, {}]],
```

Comment:
This pattern started as a bit of a hack from the pre-existing arrays of authored target drakes, saying "There could be some target drakes here, but we don't need to author them."

I feel like if there are 16 `{}`s in a row, we could say that the hack has outlived its usefulness. :)

Since the authoring json doesn't need to directly mirror the state json, should this just change to two `int`s?